CHANGELOG.md

```diff
@@ -21,19 +26,19 @@ Removed `deepFlatten` & `getHostnameFromString`.
 
 Added CommonJS version for scripts to make it possible to use them in Node.js env.
 
-To make it work we had to change [isMobile](./lib/is-mobile.js) notation. For now this helpers exports 
```

Comment:
Let's revert these changes. 

It sounds legit that you decided to “make them work”, but CHANGELOG and MIGRATION are the files that work at the exact moment of time. When the time goes, it's okay that the history becomes obsolete, links broken, etc. 

The current changes are the same as rewriting Git repo history from the very start to make a project work with modern Node.js in every commit, even those which made years ago. Nobody does it :-)